extensions/stats/plugin.cc

```diff
@@ -97,8 +105,8 @@ void map_request(IstioDimensions& instance,
   instance[request_protocol] = request.request_protocol;
   instance[response_code] = std::to_string(request.response_code);
   instance[response_flags] = request.response_flag;
-  instance[connection_security_policy] = std::string(
-      ::Wasm::Common::AuthenticationPolicyString(request.service_auth_policy));
+  instance[connection_security_policy] = to_lower(std::string(
```

Comment:
How about using this https://github.com/abseil/abseil-cpp/blob/master/absl/strings/ascii.h#L168？

Reply:
I thought we were not using absl stuff for wasm pluging?

Reply:
String related utils should be fine. they do not involve system calls.